code/modules/reagents/chemistry/machinery/pandemic.dm

```diff
@@ -140,6 +140,13 @@
 		icon_state = "mixer0"
 		src.updateUsrDialog()
 		return
+	else if (href_list["emptyeject_beaker"])
+		beaker.reagents.clear_reagents()
+		beaker:loc = src.loc
+		beaker = null
+		icon_state = "mixer0"
+		src.updateUsrDialog()
+		return
 	else if(href_list["clear"])
 		src.temp_html = ""
 		src.updateUsrDialog()
```

Comment:
No reason to use src. It's implicit. The old code is misleading.

Also, there is no reason to use a : here instead of a . for accessing the beakers loc.